test/acceptance/features/bindAppToService.feature

```diff
@@ -756,3 +756,36 @@ Feature: Bind an application to a service
             """
         Then Service Binding "binding-request-cross-ns-service" is ready
         And The application env var "BACKEND_HOST_CROSS_NS_SERVICE" has value "cross.ns.service.stable.example.com"
+
+    @negative
+    Scenario: Service binding with a missing app and configMap as a service
+        Given ConfigMap is applied
+            """
+            apiVersion: v1
+            kind: ConfigMap
+            metadata:
+                name: example
+                annotations:
+                    service.binding: path={.data},elementType=map
```

Comment:
`elementType=map` is not a part of [the spec](https://github.com/k8s-service-bindings/spec#binding-secret-generation-strategies). If we support it, we should document it.

Reply:
@pedjak I believe any annotation syntax that might have been implemented should be, in addition of being documented, ironed out by the spec proposers.